README.md

```diff
@@ -356,5 +356,149 @@ function `bodn2c`, and wants the vector version if it exists (but it doesn't!),
+spice_cell = SpiceCell(typeno=SPICE_CELL_DOUBLE, size=10)
+```
+
+With these constructors, `spice_cell.card` will be set to 0 and there will initially be
+no active elements.
+
+If you already have data that you want to convert to a spice cell, you can use a simpler
+interface:
+```python
+spice_cell = SpiceCell([1, 2, 3, 4, 5])
+spice_cell = SpiceCell(np.arange(1.0, 10.0))
+spice_cell = SpiceCell(np.arange(1.0, 10.0), size=40)  # set size explicitly
+```
+If you use these constructions, Python will create a `SpiceCell` whose size is a little 
+bigger than the length of the given data. The passed first argument will be the active
+elements.
+
+Any CSPICE function that expects a `SpiceCell` as an input can also be passed an array,
+a tuple, or anything that can reasonably be converted into an appropriately typed
+`SpiceCell`:
+
+```shell
+>>> import cspyce
+>>> cspyce.wninsd(2.0, 4.9, (1.0, 3.0, 5.0, 7.0, 9.0, 11.0))
+<SpiceCell double 6/12 [ 1.   4.9  5.   7.   9.  11. ]>
+```
+
+#### Sequence-like operations
+
+`SpiceCell` offers the following methods so that they behave like Python sequences.
+
+`spice_cell[index]`, `spice_cell[index] = value`
+
+:  Gets or sets the `index`-th element of the `SpiceCell`. 
+   Must have `-len(spice_cell) ≤ index < spice_cell.size`.  As in Python, we allow
+   negative indices, but -1, -2, etc count backwards from the last active elements.
+
+`len(spice_cell)`
+: Returns the number of active elements of the `SpiceCell`.
+
+`spice_cell.append(value)`
+: If `value` is a float or integer, it is added to the `SpiceCell`'s active cells.
+If `value` is an array, all of its elements are added to the `SpiceCell`'s active cells.
+The cell's maximum size is grown if necessary. 
+
+`spice_cell.extend(values)`
+: Appends each value in `values` to the end of the `SpiceCell`'s active cells.
+Each element of `values` can be a float, an integer, or an array.
+The `SpiceCell`'s maximum size is grown if necessary.
+
+`spice_cell += values`
+: Synonym for `spice_cell.extend(values)`
+
+`spice_cell.clear()`
+: Make all elements inactive. Same as `spice_cell.card = 0`. 
+
+`iter(spice_cell)` 
+: Iterates through the active elements of `spice_cell`.
+Because of this iterator, `list(spice_cell)`, `set(spice_cell)`, and `tuple(spice_cell)` 
+work as expected.
+You can also write a for loop `for item in spice_cell: ...` to iterate through
+the active elements.
+
+#### SpiceCell-specific operations.
+
+`SpiceCell` also has the following methods and properties specific to a `SpiceCell`.
+
+`spice_cell.size`
+: Gets the current maximum size of the `SpiceCell`.
+
+`spice_cell.size = value`
+: Grows or shrinks the maximum size of the `SpiceCell`.
+If the number of active elements is greater than `value`, it is reduced to `value`.
+This operation is a `cspyce` enhancement that does not exist in CSPICE.
+
+`spice_cell.card`
+: Same as `len(spice_cell)`.  (Also known as the "cardinality"; hence the name).
```

Comment:
```suggestion
: Same as `len(spice_cell)`.  (Also known as the "cardinality", hence the name).
```

Reply:
Semicolon is not correct grammar here. A semicolon should start a complete new sentence. The original comma I suggested is correct.

Reply:
I'm sorry, I'm not going to let Mrs. Harper, my 11th grade English teacher,  fail me even though she's been dead for twenty years.

I typed  - What punctuation to use before "hence" - to Google. 

1. Use a semicolon, not a comma, to join two independent clauses separated by a conjunctive adverb (such as however). The conjunctive adverbs include accordingly, consequently, hence, however, moreover, otherwise, therefore, and thus.

Reply:
Hmm.  But it's not really an independent clause, so you're probably right.